phpseclib/Crypt/Base.php

```diff
@@ -2001,11 +2057,48 @@ function _createInlineCryptFunction($cipher_code)
      * for which $mode the lambda function was created.
      *
      * @access private
-     * @return &Array
+     * @return Array &$functions
      */
     function &_getLambdaFunctions()
     {
         static $functions = array();
         return $functions;
     }
+
+    /**
+     * Generates a digest from $bytes
+     *
+     * @see _setupInlineCrypt()
+     * @access private
+     * @param $bytes
+     * @return String
+     */
+    function _trapdoor($bytes)
```

Comment:
I think this is a bit mincing words but, in-so-far as new names go...  how about `_generateGUID` where GUID = globally unique identifier?

That's what this function is essentially doing. It creates loop-unrolled dynamic functions that are specific to the key / block size / number rounds, etc, being used.

Rather than use the key as an index to the array, however, a hash of the key was used.

Here's what it's doing now:

```php
$code_hash = md5(str_pad("Rijndael, {$this->mode}, {$this->block_size}, ", 32, "\0") . implode(',', $w));
```

What it does after this PR:

```php
        $code_hash = "Crypt_Rijndael, {$this->mode}, {$this->Nr}, {$this->Nb}";
        if ($gen_hi_opt_code) {
            $code_hash = str_pad($code_hash, 32) . $this->_trapdoor($this->key);
        }
```

I'm just asking now because I don't want to go make a commit to change the name to something and for you to then come back and say you think that's not a good name either lol. I'd rather just figure out what a good name is and *then* make the change.